meta-mender-core/classes/mender-part-images.bbclass

```diff
@@ -42,19 +42,6 @@ IMAGE_NAME_SUFFIX = ""
 # Block storage
 ################################################################################
 
-def mender_wic_exclude_path_options(rootfs, exclude_path):
-    exclude_path = exclude_path.strip()
-
-    existing_exclude_paths = ""
-    for path in exclude_path.split():
-        if os.path.lexists(os.path.realpath(os.path.join(rootfs, path))):
-           existing_exclude_paths += path + " "
-
-    if existing_exclude_paths:
-        return "--exclude-path %s" % existing_exclude_paths
-    else:
-        return ""
-
```

Comment:
Does this mean that we are no longer excluding `/boot` and `/data` from the rootfs?

Reply:
`rawcopy` uses the `ext4` file, where they should already be removed. The difference is that `wic` is not doing that job anymore.

Reply:
That is even better :)